content/en/blog/_posts/2023-08-21-job-update-post.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Improved failure handling for Jobs"
+date: 2023-08-21
+slug: kubernetes-1-28-jobapi-update
+---
+
+**Authors:** Kevin Hannon (G-Research), Michał Woźniak (Google)
+
+This blog discusses two new features in Kubernetes 1.28 to improve Jobs for batch
+users: [Pod replacement policy](/docs/concepts/workloads/controllers/job/#pod-replacement-policy)
+and [Backoff limit per index](/docs/concepts/workloads/controllers/job/#backoff-limit-per-index).
+
+These features provide improvements to the handling of pod failures in Job, which
+continue the effort started by the
+[Pod failure policy](/docs/concepts/workloads/controllers/job/#pod-failure-policy)
+feature.
+
+## Pod replacement policy {#pod-replacement-policy}
+
+By default, when a pod enters a terminating state (e.g. due to preemption or
+eviction), a replacement pod is created immediately, and both pods are running
+at the same time. In API terms, a pod is considered terminating when it has a
+`deletionTimestamp` and it has a phase `Pending` or `Running`.
+
+This is problematic for some popular machine learning frameworks, such as
```

Comment:
The initial "This" in the paragraph might be vague for the reader. I suggest:

The scenario where no Pods are running at a given time is problematic for some popular...

Reply:
Done, just "no Pods" -> "two Pods" :)